examples/01_plotting/plot_surf_stat_map.py

```diff
@@ -0,0 +1,58 @@
+"""
+Demonstrate basic surface plotting with plot_surf_stat_map
+==========================================================
```

Comment:
> The parcellations are provided with the FS distribution, so I guess we could
> use them?

Well, we cannot rely on people having FS installed, so we would need to
provide downloaders, and for this we would need to check that there
exists URLs where we can download them.

> median value instead of the mean. But I thought this would go into a separate
> function (something similar to plot_roi?) and thus separate example.

I agree. Let's do this later.

> So maybe I should look into creating an example map from the nki
> dataset?

Let's worry about this later.